update  manage-resources documentation (#33777)
kubernetes/website: content/en/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace.md

[default CPU request and limit](/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace/)
from the LimitRange for this namespace.

> At this point, your Pod might be running or it might not be running. Recall that a prerequisite for

Reviewer: ```suggestion
At this point, your Pod may or may not be running. Recall that a prerequisite for
```
